eZ/Publish/Core/MVC/Symfony/Security/User/Provider.php

```diff
@@ -118,6 +118,6 @@ public function supportsClass($class)
      */
     public function loadUserByAPIUser(APIUser $apiUser)
     {
-        return new User($apiUser);
+        return new User($apiUser, array('ROLE_USER'));
```

Comment:
nitpick, but you can use `[]` now, otherwise @alongosz will not be able to read it :)